docs/man_pages/publishing/appmanager-livesync.md

```diff
@@ -0,0 +1,42 @@
+appmanager livesync
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder appmanager livesync [<Platforms>]`
+
+Publish new patch of the project to AppManager. 
+If you have not published a LivePatch-enabled major version of this app before, you will not be able to distribute a LivePatch for it.
+If you have not enabled LivePatch plugin for your project, it will be enabled for release configuration.
+In case you do not specify a platform, the patch will be published for all platforms (in case you confirm the prompt dialog).
+
+`<Platforms>` can be one or more of the supported platforms: Android, iOS, WP8 (casing is not mandatory).
+
+<% if(isMobileWebsite) { %>
+This command is not applicable to Mobile Website projects.
+<% } %>
+
+<% if(isHtml) { %> 
+
```

Comment:
### Prerequisites

* You must have published a major version of your app, enabled for AppManager LiveSync. To create a new major version enabled for AppManager LiveSync, run \`$ appbuilder appmanager upload \<Platform\>\`